### DISCUSSION TITLE
Difficulty Importing JavaFX modules (jar files) into Py5 (JPype user)

### DISCUSSION THREAD

**Discussion by vsquared** [jpype-project/jpype#1188] [General]
We are having difficulty importing seven JavaFX modules (jar files) into Py5 which is reliant upon JPype.  We are using a Thonny editor and get the following error message which originated in the  _JPype Imports Module_:
```
ImportError: Unable to import 'javafx.scene.control.Button' due to initializer error
```
Thanks in advance for any help in stopping the error.

**Comment by Thrameos** [comment]
This isn't really a JPype issue so much as JavaFX though your report is to scarce on details for me to be sure.   JavaFX libraries are divided into a java portion and a native library.   My guess is that something in the java class or library path is incorrect such that the import is failing.   If one of the to halves is missing or unable to load you should get an error like that.   I have run JavaFX on RPi5 and know that JavaFX with the right combination can be run.  The issue I faces was getting to the actual error of which library file was missing.   Java doesn't do a lot of reporting on issues like this.
